shell/common/vsync_waiters_test.h

```diff
@@ -4,8 +4,8 @@
 
 #define FML_USED_ON_EMBEDDER
 
-#ifndef FLUTTER_SHELL_COMMON_TEST_VSYNC_WAITERS_H_
-#define FLUTTER_SHELL_COMMON_TEST_VSYNC_WAITERS_H_
+#ifndef FLUTTER_SHELL_COMMON_VSYNC_WAITERS_TEST_H_
```

Comment:
Don't we typically use the "*_test.cc" filename for things with tests in them?

Reply:
Yes, that is what this PR is changing it to.